src/librustc/traits/mod.rs

```diff
@@ -650,12 +650,12 @@ pub fn normalize_and_test_predicates<'a, 'tcx>(tcx: TyCtxt<'a, 'tcx, 'tcx>,
 /// Given a trait `trait_ref`, iterates the vtable entries
 /// that come from `trait_ref`, including its supertraits.
 #[inline] // FIXME(#35870) Avoid closures being unexported due to impl Trait.
-pub fn get_vtable_methods<'a, 'tcx>(
+pub fn vtable_methods<'a, 'tcx>(
```

Comment:
This need not be public anymore.

Reply:
🤦‍ Yep, Another one that I'll do real quick.